packages/notifi-dapp-example/src/context/NotifiContext.tsx

```diff
@@ -15,40 +17,86 @@ export const NotifiContextProvider: React.FC<PropsWithChildren> = ({
   children,
 }) => {
   const { wallets, selectedWallet } = useWallets();
+  const { wallets: injectiveWallets, selectedWallet: injectiveSelectedWallet } =
+    useInjectiveWallets();
 
   if (
-    !selectedWallet ||
-    !wallets[selectedWallet].walletKeys ||
-    !wallets[selectedWallet].signArbitrary
+    (!selectedWallet ||
+      !wallets[selectedWallet].walletKeys ||
+      !wallets[selectedWallet].signArbitrary) &&
```

Comment:
Do we need to use the `&&` statement? or we can simply chain all those conditions with `||` ?

Reply:
yes, you are right, we can chain all the conditions with 'or'